### PR TITLE
Add postcss sorting config and stylelint properties order rule

### PR DESCRIPTION
**Pull request by dzhebeuilovo**

### CODE REVIEW

**Comment by EmilShakirov** [comment]
👍

File: config/postcss-cli.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
+module.exports = {
+  "postcss-sorting": {
+    "empty-lines-between-media-rules": 1,
+    "empty-lines-between-children-rules": 1,
+
```
Comment (maratfakhreev) [inline config/postcss-cli.js]: Pls remove empty line

File: config/stylelint.js [hunk 2]
```diff
@@ -22,6 +22,133 @@ module.exports = {
     'selector-list-comma-newline-after': 'always-multi-line',
     'selector-no-id': true,
     'string-quotes': 'double',
-    'value-no-vendor-prefix': true
+    'value-no-vendor-prefix': true,
+    'declaration-block-properties-order': [
```
Comment (maratfakhreev) [inline config/stylelint.js]: Try to import sorting config and write smth like: `declaration-block-properties-order: config["postcss-sorting"]["sort-order"][3];`

File: package.json [hunk 3]
```diff
@@ -10,6 +10,7 @@
   },
   "scripts": {
     "stylelint": "stylelint ./app/**/*.css --config ./config/stylelint.js",
+    "stylesort": "postcss -u postcss-sorting -c ./config/postcss-cli.js -r ./app/{**,**/**}/*.css ./app/{**,**/**}/*.css",
```
Comment (maratfakhreev) [inline package.json]: Why twice `./app/{**,**/**}/*.css`?

Comment (dzhebeuilovo) [inline package.json]: Thx, removed

File: config/stylelint.js [hunk 4]
```diff
@@ -1,3 +1,5 @@
+var postcssCLIConfig = require('./postcss-cli');
```
Comment (maratfakhreev) [inline config/stylelint.js]: Use `const`

**Comment by maratfakhreev** [comment]
🆗

**Pull request merged**